### PR TITLE
[ticket/12577] Use a lazy service for the password manager

### PR DESCRIPTION
**Pull request by Nicofuma**
https://tracker.phpbb.com/browse/PHPBB3-12577

PHPBB3-12577

### CODE REVIEW

File: build/build.xml [hunk 1]
```diff
@@ -310,6 +310,18 @@
...
+		<delete dir="${dir}/vendor/ocramius/proxy-manager/docs" />
+		<delete dir="${dir}/vendor/ocramius/proxy-manager/build" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/.gitignore" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/.travis.yml" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/CONTRIBUTING.md" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/phpdox.xml.dist" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/phpmd.xml.dist" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/phpunit.xml.dist" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/README.md" />
+		<delete file="${dir}/vendor/ocramius/proxy-manager/UPGRADE.md" />
```
Comment (marc1706) [inline build/build.xml]: The folder examples and html-docs as well as the files .scrutinizer.yml, .travis.install.sh, index.html, proxy-manager.png, proxy-manager.svg, and STABILITY.md also seem unneeded to me?

File: build/build.xml [hunk 2]
```diff
@@ -362,6 +374,12 @@
 		<delete file="${dir}/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/README.md" />
 		<delete file="${dir}/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/phpunit.xml.dist" />
 
+		<delete dir="${dir}/vendor/symfony/proxy-manager-bridge/Symfony/Bridge/ProxyManager/Tests" />
+		<delete file="${dir}/vendor/symfony/proxy-manager-bridge/Symfony/Bridge/ProxyManager/.gitignore" />
```
Comment (marc1706) [inline build/build.xml]: Please make sure to also remove the .git folders. See related ticket for current develop: https://tracker.phpbb.com/browse/PHPBB3-13619

**Comment by marc1706** [comment]
Lock file seems to be out of date.

**Comment by marc1706** [comment]
Please address these comments and rebase onto current master branch.

**Comment by lavigor** [comment]
@Nicofuma, I wonder why this PR affected all the tickets in the commit messages.
Those tickets are not related to this PR, but they were automatically opened after this PR was closed.

**Pull request closed without merging**